fe/fe-core/src/main/java/org/apache/doris/nereids/trees/plans/logical/LogicalExcept.java

```diff
@@ -34,7 +35,7 @@
 /**
  * Logical Except.
  */
-public class LogicalExcept extends LogicalSetOperation {
+public class LogicalExcept extends LogicalSetOperation implements PropagateFD {
```

Comment:
why not add all columns as a unique cluster into FD?

Reply:
Yes, I forgot about that.